helmfile.d/values/networkpolicies/workload/fluentd.yaml.gotmpl

```diff
@@ -0,0 +1,59 @@
+{{- $netpol := .Values.networkPolicies -}}
+
+policies:
+  fluentd-system:
+    forwarder-to-opensearch:
+      podSelectorLabels:
+        app.kubernetes.io/instance: fluentd-forwarder
+      egress:
+        - rule: egress-rule-sc-ingress
+    aggregator-from-user:
+      podSelectorLabels:
+        app.kubernetes.io/instance: fluentd-aggregator
+      ingress:
+        - name: fluentd-elasticsearch
+          peers:
+            - namespaceSelectorLabels:
+                kubernetes.io/metadata.name: fluentd
+              podSelectorLabels:
+                app.kubernetes.io/instance: fluentd-elasticsearch
+          ports:
+            - tcp: 24224
+  fluentd:
+    all-deny:
+      podSelectorLabels: {}
+      egress: {}
+      ingress: {}
+
+    forwarder:
+      podSelectorLabels:
+        app.kubernetes.io/name: fluentd-elasticsearch
+      egress:
+        - rule: egress-rule-apiserver
+        - rule: egress-rule-dns
+        - rule: egress-rule-sc-ingress
+        - name: fluentd-aggregator
+          peers:
+            - namespaceSelectorLabels:
+                kubernetes.io/metadata.name: fluentd-system
+              podSelectorLabels:
+                app.kubernetes.io/instance: fluentd-aggregator
+          ports:
+            - tcp: 24224
+      ingress:
+        - rule: ingress-rule-prometheus
+          ports:
+            - tcp: 24231
+    extra-plugin:
+      podSelectorLabels:
+        app.kubernetes.io/name: fluentd-elasticsearch
+      egress:
+        - name: fluentd-extra-plugin
+          peers:
+            {{- range $netpol.fluentd.extraPlugin.ips }}
+            - cidr: {{ . }}
+            {{- end }}
+          ports:
+            {{- range $netpol.fluentd.extraPlugin.ports }}
+            - tcp: {{ . }}
+            {{- end }}
```

Comment:
Maybe this should be a conditional rule in the regular one? Also just check what happens when ips and ports are empty so it don't render a allow all rule, I don't remember if we guard against it in the generator :sweat_smile: